client/src/components/infoDrawer/infoDrawer.js

```diff
@@ -106,15 +122,101 @@ const singleValueCategoriesPlaceholder = Array.from(Array(NUM_CATEGORIES)).map(
   }
 );
 
+const renderDOILink = (type, doi, skeleton) => {
+  return (
+    doi && (
+      <>
+        <H3 className={skeleton ? Classes.SKELETON : null}>{type}</H3>
+        <p className={skeleton ? Classes.SKELETON : null}>
+          <a href={doi} target="_blank" rel="noopener noreferrer">
+            {doi}
+          </a>
+        </p>
+      </>
+    )
+  );
+};
+
+const affiliations = [];
+const renderContributors = (contributor) => {
+  const { institution, email, name } = contributor;
+  let index;
+  if (institution) {
+    index = affiliations.indexOf(institution) + 1;
+    if (index === 0) {
+      index = affiliations.push(institution);
+    }
+  }
+  return (
+    <span key={name}>
+      {name}
+      {email && `(${email})`}
+      <sup>{index}</sup>
+    </span>
+  );
+};
+
 const InfoFormat = ({
```

Comment:
Should we extract `InfoFormat` into its own file and import it for `infoDrawer` to use? I think that'd help with the growing file size!